### PR TITLE
[tests] tidy up wallet_importmulti.py

### PR DESCRIPTION
**Pull request by amitiuttarwar**
Cherry picks un-merged commits from #14952, which "fixes review comments from @ryanofsky here: https://github.com/bitcoin/bitcoin/pull/14886#pullrequestreview-183772779"

### CODE REVIEW

**Comment by Sjors** [comment]
Concept ACK, will review later.

**Review by promag — COMMENTED** [review]
@amitiuttarwar side note, looks like you haven't added your email to github?

File: test/functional/test_framework/wallet_util.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018 The Bitcoin Core developers
```
[the remark this answers is not in the document]
Comment (MarcoFalke) [reply test/functional/test_framework/wallet_util.py]: This was authored in 2018 (according to the git author date), so I guess it is fine.

**Comment by sipa** [comment]
Concept ACK

**Review by jnewbery — COMMENTED** [review]
Looks great. Thanks for picking this up @amitiuttarwar!

There were a couple of review comments from @practicalswift in #14952 which weren't addressed in the base PR #14565, so you could fix them in this PR if you wanted:

- wallet_importmulti.py L52 (`observed_warnings = result[0]['warnings']` is underindented)
- the default argument for `warnings` in `test_importmulti()` should be `None`, not `[]` (see 'Mutable Default Arguments' in https://docs.python-guide.org/writing/gotchas/)

I've also added one small nit of my own.

tested ACK 2d5f1ea2e3f9629d313f1d2aff94a15ef8e1128a

**Comment by Sjors** [comment]
tACK 2d5f1ea

Fixing nits is nice, but adding tests to #14491 will be easier once this is merged (cc @MeshCollider).

**Comment by MarcoFalke** [comment]
I am merging this, because it has some review and the nits are only about whitespace in touched lines or additional cleanup in unrelated lines. Please fix them up separately if anyone feels like it.

**Pull request merged**